data/darktableconfig.xml.in

```diff
@@ -2124,6 +2124,19 @@
     <shortdescription>border around image in darkroom mode</shortdescription>
     <longdescription>process the image in darkroom mode with a small border. set to 0 if you don't want any border.</longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>darkroom/ui/iso12464_border</name>
+    <type>float</type>
```

Comment:
can't we define some min / max here to avoid crazy values possible breaking the display?

Reply:
Did that